addon/addon/models/sprite-tree.ts

```diff
@@ -23,7 +23,7 @@ export interface IContext {
   appendOrphan(spriteOrElement: Sprite): void;
   clearOrphans(): void;
   args: {
-    use?(changeset: Changeset): AnimationDefinition;
+    use?(changeset: Changeset): AnimationDefinition | Promise<void>;
```

Comment:
The `Promise<void>` seems like it's based on the old system/api. We won't support this in the future, but right now a lot of the demos are still using the old API. Just something to keep in mind I guess (as we're unlikely to refactor all demos immediately).